vault/external_tests/api/kv_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package api
+
+import (
+	"context"
+	"testing"
+
+	"github.com/hashicorp/vault/helper/testhelpers/minimal"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/hashicorp/vault/api"
+)
+
+func TestKVV1Get(t *testing.T) {
+	t.Parallel()
+
+	cluster := minimal.NewTestSoloCluster(t, nil)
+
+	cluster.Start()
```

Comment:
FYI this is a no-op for any use of TestCluster (including NewTestCluster), the Start method exists only because we haven't got around to removing all uses of it.